adafruit_logging.py

```diff
@@ -117,6 +113,8 @@ def write(self, buf: str) -> int:
 for __value, __name in LEVELS:
     globals()[__name] = __value
 
+_ROOT_LOGGER_SENTINEL = object()
```

Comment:
The idea was to make an object that was both 1) entirely unique and 2) not a string (since someone could conceivably use `""` as a label.  I didn't realize that CircuitPython wouldn't allow for objects as dictionary keys, thanks for catching that.

I think using a blank string is sufficiently "unique" though, and the sentinel to that and just add a note to the docstring for `getLogger()` that no argument *or a blank string* will bring up the root logger.